src/components/AdminAppointmentDetailsPopup.tsx

```diff
@@ -0,0 +1,502 @@
+              console.log(
+                'Successfully sent confirmation email for: ',
+                confirmationEmailData,
+              );
+            })
+            .catch((error: Error) => {
+              console.log(error);
+            });
+        })
+        .catch((error: Error) => {
+          console.log(error);
+          toast({
+            title: 'Oops, Appointment Planetary Alignment Off: Error Detected',
+            description:
+              'Planetary alignment for the appointment is slightly off due to an error.',
+          });
+        });
+    });
+  };
+
+  const renderStatusBadges = () => {
+    const statusList: string[] = [];
+    if (is_emergency) {
+      statusList.push('Emergency');
+    }
+    if (isAppointmentScheduled && !isAppointmentCancelled) {
+      statusList.push('Scheduled');
+    }
+    if (isAppointmentCancelled) {
+      statusList.push('Cancelled');
+    }
+    if (!isAppointmentScheduled && !isAppointmentCancelled) {
+      statusList.push('Waiting');
+    }
+
+    return statusList.map((status, index) => (
+      <Badge key={index} variant="secondary">
+        {status}
+      </Badge>
+    ));
+  };
+
+  const renderViewDetailsTabContent = () => (
+    <>
+      <DialogHeader>
+        <DialogTitle className="py-2 text-center text-2xl border-b">
+          <span className="w-full text-transparent bg-clip-text bg-gradient-to-r from-blue-400 to-rose-500">
+            Patient Chronicles
+          </span>{' '}
+          <br />A Peek into the Appointment
+        </DialogTitle>
+      </DialogHeader>
+      <div className="mt-4 px-2 flex gap-4">{renderStatusBadges()}</div>
+      <div className="grid gap-2 py-2 px-2">
+        <div className="flex gap-4">
+          <h3 className="font-bold">Name :</h3>
+          <p>{`${first_name} ${last_name}`}</p>
+        </div>
+        <div className="flex gap-4">
+          <h3 className="font-bold">Email: </h3>
+          <p>
+            <a
+              href={`mailto:${email}`}
+              className="no-underline transition-all hover:underline ease-in-out"
+            >
+              {email}
+            </a>
+          </p>
+        </div>
+        <div className="flex gap-4">
+          <h3 className="font-bold">Contact Number: </h3>
+          <p>
+            <a
+              href={`tel:${mobile_phone}`}
+              className="no-underline transition-all hover:underline ease-in-out"
+            >
+              {formatMobilePhone(mobile_phone)}
+            </a>
+          </p>
+        </div>
+        <div className="flex gap-4">
+          <h3 className="font-bold">Type of Appointment: </h3>
+          <p>
+            {appointment_type
+              ? `${appointment_type[0].toUpperCase()}${appointment_type.slice(
+                  1,
+                )}`
+              : 'N/A'}
+          </p>
+        </div>
+        <div className="flex flex-col gap-1">
+          <h3 className="font-bold">Description: </h3>
```

Comment:
Maybe this can also be conditionally rendered: if there are no details given for the description, then we can reduce the clutter by not rendering the Description field